src/calculateRentalCost.js

```diff
@@ -28,7 +28,10 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const priceOfDay = 40;
+  const discount = (days >= 7) ? 50 : (days >= 3) ? 20 : 0;
+
```

Comment:
You may also name variables for 50, 20, cause it's look like "magic numbers". p5